### ISSUE TITLE
FBX model import does not support relative path texture imports

### ISSUE THREAD

**Issue by philpax** [AmbientRun/Ambient#643]
I tried to use https://sketchfab.com/3d-models/phoenix-bird-844ba0cf144a413ea92c779f18912042 within my game, but the resulting mesh was untextured:

<img width="699" alt="image" src="https://github.com/AmbientRun/Ambient/assets/707827/e6185e9b-6576-451c-9bd6-6cd44e51cf59">

The Sketchfab FBXes contain the textures outside the FBX, with the FBX having relative paths to the textures. Our import process should take this into account and pull in the textures.

**Comment by FredrikNoren** [comment]
It's possible to manually set the materials/textures like this: https://ambientrun.github.io/Ambient/reference/asset_pipeline.html#a-more-complex-model-example But that can probably be made more ergonomic (need to run build and then look in the build folder to find out what the material names are).

Might also be possible to automatically find the textures if the fbx file contains textures with relative paths; can't remember if we implemented this or not.

**Comment by philpax** [comment]
Yeah, I'm guessing that it's a relative path. Having a quick look at `model_import` suggests that we don't do this; we'll probably need to implement this at some point. I'll update the issue.
